core/src/col_metadata.rs

```diff
@@ -274,7 +274,7 @@ impl ObjectSchema {
 
```

Comment:
No real changes in this file, just fallout from the refactor and addition of error messages to the ResultSetSchema error